.devcontainer/Dockerfile

```diff
@@ -45,7 +42,6 @@ ENV HISTFILE="/home/coder/.cache/._bash_history"
 ENV AWS_ROLE_ARN="arn:aws:iam::279114543810:role/nv-gha-token-sccache-devs"
 ENV SCCACHE_REGION="us-east-2"
 ENV SCCACHE_BUCKET="rapids-sccache-devs"
-ENV SCCACHE_S3_USE_PREPROCESSOR_CACHE_MODE=true
```

Comment:
Seems like we have some additional changes in here?

Reply:
I actually verified that this file is consistent with release/25.12 as well as with main. Not sure why git is showing this diff.

Reply:
have you rebased this branch?